drivers/interrupt_controller/Kconfig.it8xxx2

```diff
@@ -0,0 +1,13 @@
+# Kconfig - configuration options
+#
+# Copyright (c) 2015 ITE Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+config ITE_IT8XXX2_INTC
+	bool "ITE_IT8XXX2 Interrupt controller"
+	depends on ( SOC_IT83202|| SOC_IT81302|| SOC_IT8XXX2)
+	help
+	 Configures the maximum number of clients allowed per shared
+	 instance of the shared interrupt driver. To conserve RAM set
+	 this value to the lowest practical value.
```

Comment:
Set a default? How do you know what to set this to? I think the help should be more explicit